### DISCUSSION TITLE
Compare segments

### DISCUSSION THREAD

**Discussion by Piff91** [mytourbook/mytourbook#1347] [Q&A]
Hello, I have a fairly recurring problem when comparing segments.
The compared segment does not match the reference segment. It differs by distance, so the time is not correct.
There is a solution?
How does the geographic comparison work?

**Comment by wolfgang-ch** [comment]
> Hello, I have a fairly recurring problem when comparing segments. The compared segment does not match the reference segment. It differs by distance, so the time is not correct. There is a solution? 

Can you describe or show in screenshots what is wrong and what should be the correct result


>How does the geographic comparison work?

see https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tool-compare-tours-by-geo#how-it-works

**Comment by Piff91** [comment]
The problem is that the segment is larger or smaller than the reference segment.
When I created the segment I perhaps traveled a few more meters between the beginning and the end. Now that I'm faster, I get to the benchmark but the segment isn't over yet. It follows that the times compared are not real.
I leave the screenshot attached.
![Screenshot 2024-05-16 211716](https://github.com/mytourbook/mytourbook/assets/131846053/d9e3a1a3-a3da-4360-ac1a-c0d5070e3352)
![Screenshot 2024-05-16 212434](https://github.com/mytourbook/mytourbook/assets/131846053/b71af955-c8d5-4778-aadd-cf05e1d020a1)
On 05/14 I took 27min, but from the comparison it took 34min
![Screenshot 2024-05-16 213239](https://github.com/mytourbook/mytourbook/assets/131846053/873e068a-f9b2-4b09-b24a-ee73043a8e55)
There is a difference of 80m

**Comment by wolfgang-ch** [comment]
I'm a bit confused, for which tour comparison is this issue, elevation or geo compare?

**Comment by Piff91** [comment]
Elevation

Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Piff91** [comment]
the problem is with elevation. I've never used geo, just asked how it worked.
Thanks

Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by wolfgang-ch** [comment]
The compare result can be adjusted https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tool-compare-tours#adjust-compare-result

**Comment by Piff91** [comment]
Sorry, I already tried to adjust the comparison. The problem is that if I match the beginning of the segment, the end of the segment is incorrect.
From what I understand, the length of the segment does not match the compared one. Is there a way to adapt it?
Moving left or right only helps me adjust the beginning or end but not both.
________________________________
Da: Wolfgang Schramm ***@***.***>
Inviato: lunedì 20 maggio 2024 16:17
A: mytourbook/mytourbook ***@***.***>
Cc: Piff91 ***@***.***>; Author ***@***.***>
Oggetto: Re: [mytourbook/mytourbook] Compare segments (Discussion #1347)


The compare result can be adjusted https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/tools/tool-compare-tours#adjust-compare-result

**Reply by wolfgang-ch** [reply]
The x-axis of the chart should display the distance and not the time

**Comment by Piff91** [comment]
In fact I always use it in km. I don't know if you were able to notice that in the photos I attached to you the other day, if I adjust the segment with the mouse and put it where the beginning should be, the end of the segment is 80m beyond. So I deduce that the original segment is longer than 80m.
I also believe that there will always be minimal variation, if you zigzag uphill one time and do a linear trail the next time.
The question is whether it can be fixed.
________________________________
Da: Wolfgang Schramm ***@***.***>
Inviato: lunedì 20 maggio 2024 18:00
A: mytourbook/mytourbook ***@***.***>
Cc: Piff91 ***@***.***>; Author ***@***.***>
Oggetto: Re: [mytourbook/mytourbook] Compare segments (Discussion #1347)


The x-axis of the chart should display the distance and not the time

**Comment by wolfgang-ch** [comment]
> I also believe that there will always be minimal variation, if you zigzag uphill one time and do a linear trail the next time.
The question is whether it can be fixed.

Currently there is no tool in MT to adjust these distance differences however you can manually delete time slices in the tour editor
